### ISSUE TITLE
Permission to feature PnP-Mass in the DeepInverse documentation

### ISSUE THREAD

**Issue by Tmodrzyk** [hubert-leterme/weaklensing_uq#1]
Hi @hubert-leterme,

We've noticed that your paper uses Deepinv to implement the method. 
We're very happy to see the library used in other application fields ! 

We’re creating a gallery showcasing research built with DeepInverse on our documentation website (https://deepinv.org/) and would love to feature your work. Would you be comfortable with us including it?

If so, we would credit both the paper and your code, add a small description of your work, and if possible, include an illustration figure taken from one of the two figures.
The description could be something like: "PnPMass is a new plug-and-play approach to estimate the dark matter distribution from noisy weak-lensing shear measurements."

Of course we would be happy to incorporate any edits or remove the entry later on your request.

Many thanks !

**Comment by hubert-leterme** [comment]
Dear @Tmodrzyk,

Thank you very much for your interest in our paper. We would definitely be comfortable with you featuring this work in the DeepInverse documentation! Feel free to use any plot you like from the paper.

I have a few remarks:

- I am currently working on polishing our implementation, with cleaner file structure and documentation. In particular, the current version of our package wlmmuq uses an outdated version of DeepInverse (0.2.2), and therefore needs an upgrade. I'll let you know as soon as the new version is ready.

- I talked with Julián Tachella a while ago regarding a potential integration of our method within the deepinv package directly (including our custom data fidelity class, the weak-lensing forward operator, the loss function to train order-2 moment networks for uncertainty quantification...). He told me that it could be an interesting contribution to the library, and invited me to create a pull request. Would you still be interested in such a collaboration?

Feel free to contact me if you need any additional information.

Best,
Hubert

**Comment by Tmodrzyk** [comment]
Hi again !
Thanks for your response, I might ping you later on the related PR to make sure that the showcase suits you: https://github.com/deepinv/deepinv/pull/1412.

Regarding the broader collaboration, it would be awesome and we are definitely still interested ! The most natural first step is usually the implementation / integration of the forward operator. I can walk you through our contribution pipeline if you need help. 😄

**Comment by hubert-leterme** [comment]
That would be great. I'll let you know when I'll start working on it, to make sure I correctly follow the contribution pipeline.
